jekyll/_cci2/configuration-reference.md

```diff
@@ -648,7 +660,8 @@ when | N | String | [Specify when to enable or disable the step](#the-when-attri
 
 Each `run` declaration represents a new shell. It is possible to specify a multi-line `command`, each line of which will be run in the same shell:
 
-``` YAML
+``` yaml
+version: 2.1
```

Comment:
I think it looks a bit odd having the version and then straight into `- run:` without the job and step keys?

Reply:
Same for other partial examples below ...